src/main/resources/templates/footer.ftl

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
maybe add some info about yourself in footer

Reply:
fixed